libs/libzauth/libzauth-c/src/lib.rs

```diff
@@ -130,10 +130,11 @@ pub extern fn zauth_token_type(t: &ZauthToken) -> ZauthTokenType {
     From::from(t.0.token_type)
 }
 
-#[no_mangle]
-pub extern fn zauth_token_time(t: &ZauthToken) -> c_long {
-    t.0.timestamp
-}
+// Commented out, looks unused, and causing portability issues with ia32.
+//#[no_mangle]
+//pub extern fn zauth_token_time(t: &ZauthToken) -> c_long {
+//    t.0.timestamp
+//}
```

Comment:
while this particular function might indeed not be used by the [nginz zauth module](https://github.com/wireapp/wire-server/blob/develop/services/nginz/third_party/nginx-zauth-module/zauth_module.c), I have a question: Does the underlying issue with the timestamp remain in ia32 upon [token verification](https://github.com/wireapp/wire-server/blob/develop/libs/libzauth/libzauth/src/zauth.rs#L138), i.e. in [is_expired](https://github.com/wireapp/wire-server/blob/develop/libs/libzauth/libzauth/src/zauth.rs#L166-L169)?

Can you confirm you can compile nginz, and in demo-mode nginz correctly handles expired vs non-expired tokens under ia32? If not, commenting out this code won't help much. If this is just an FFI portability issue, but verification happens as expected under nginz, I'd prefer removing the code, rather than commenting it out.

Reply:
I can confirm that i can compile / launch nginz, but i have no idea how to perform the test you just aluded to.

Reply:
it looks like the code you've pointed to is using 64 bit integers, even on ia32, so should not have any problems until December 4th, 292,277,026,596 AD.